airflow-core/src/airflow/triggers/deadline.py

```diff
@@ -49,26 +49,25 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
         from airflow.models.deadline import DeadlineCallbackState  # to avoid cyclic imports
 
         try:
-            callback = import_string(self.callback_path)
             yield TriggerEvent({PAYLOAD_STATUS_KEY: DeadlineCallbackState.RUNNING})
+            callback = import_string(self.callback_path)
 
-            # TODO: get airflow context
-            context: dict = {}
-
-            result = await callback(**self.callback_kwargs, context=context)
-            log.info("Deadline callback completed with return value: %s", result)
```

Comment:
The triggerer already emits an info log when the TriggerEvent is yielded so we don't need an extra log here